.project

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I still think is helpful, we would still have to add classpaths manually to the project though.